medical/models/medical_physician.py

```diff
@@ -111,102 +107,3 @@ def create(self, cr, uid, vals, context=None):
         vals['groups_id'] = [(6, 0, group_ids)]
```

Comment:
Ahhh thanks @JonathanNEMRY - I knew I recognized this variable name from somewhere. Will remove here